packages/pass-style/src/far-class-instances.js

```diff
@@ -0,0 +1,19 @@
+import { Far } from './make-far.js';
+
+/**
+ * Classes whose instances should be Far objects should inherit from
```

Comment:
"should" … why? Please expand with what risk the developer is taking if they don't heed the advice.

Reply:
Good point. Fixed to

> Classes whose instances should be Far objects may find it convenient to inherit from this base class.

Done.